monero-outreach-docs/translations/es/archive/triptych_es.md

```diff
@@ -0,0 +1,43 @@
+# Triptcyh: Un nuevo algoritmo que protege a los usuarios de Monero
+
+**“Monero oculta al emisor, al receptor y la cantidad en una transacción. Triptych (pronunciado trip-tic) es una actualización para Monero que es más rápida, ligera y privada.” - 1 de septiembre, 2020**
+
+Detrás de los detalles en cómo Monero oculta al emisor, al receptor y la cantidad en una transacción, existe un funcionamiento en constante evolución. Este artículo habla sobre Triptych (pronunciado trip-tic), una tecnología nueva de Monero que oculta mejor al emisor usando menos espacio de la cadena de bloques y un menor tiempo en el procesamiento.
+
+### _Firmas de anillo_
+
+Ocultar a quien envía en Monero toma más que tan solo proteger direcciones. Todo dentro de la cadena de bloques de Monero se representa como transacciones salientes o outputs, llamados TXO. Cada TXO está etiquetado con un número de identificación único o ID, que se utiliza una sola vez, llamada llave pública o dirección oculta (stealth address). Con tan solo observar la cadena de bloques, las llaves públicas TXO no pueden ser enlazadas con direcciones de Monero. Sin embargo, las llaves públicas sí necesitan protección. Sería inaceptable para el flujo de Monero que estás llaves fuesen transparentes teniendo en cuenta que pudieran existir técnicas externas a la cadena de bloques que conectaran esas llaves a personas.
```

Comment:
A correction here according to the [RAE](https://www.rae.es/espanol-al-dia/cambio-de-la-o-disyuntiva-en-u-0):

_Todo dentro de la cadena de bloques de Monero se representa como transacciones salientes **u** outputs..._

Reply:
Fixed :)